armi/cli/checkInputs.py

```diff
@@ -140,4 +139,4 @@ def invoke(self):
         )
 
         if any(t[2] != "PASSED" or t[3] != "PASSED" for t in table):
-            sys.exit(-1)
+            runLog.warning(f"Check did not pass with flying colors")
```

Comment:
@john-science what do you think about this? I really don't think the fail-fail is necessary. This also allows me to address the issues with `armiRun.yaml` separately.

@keckler @albeanth do you see any downstream issues with not exiting? This is only the CLI that does check-inputs, not the code itself

Reply:
I don't have any issue. I've never used this feature 😄

Reply:
1. I guess the question is: do we want people to be able to use this "check inputs" as a safety check? Like, to stop a run?
2. What about making it a `runLog.error()`?  That won't exit the program, but it make it very visible.

Reply:
I am leaning towards `runLog.error`. 

When someone is programatically using `checkInputs`, they are looking for whether or not there are any issues tallied:
``` python
return not any(inspectorIssues)
```

When someone is using it from the command line, a table prints out:
```
Finding potential settings files matching ['armi/tests/armiRun.yaml'].
Checking for valid settings files.
loaded armi/tests/armiRun.yaml
----------------------------------------------------------------  -------  -----------  --------------------------
                                                                         case     can start    input is self consistent

       ----------------------------------------------------------------  -------  -----------  --------------------------
       C:\Users\aopotowsky\codes\nala\framework\armi\tests\armiRun.yaml  armiRun  UNKNOWN      PASSED
       ----------------------------------------------------------------  -------  -----------  --------------------------
```
So I'm just not convinced there needs to be anything more because the following step would be user-decided on the command line

Reply:
Cool. As long as you have the workflow in your mind.

If the user just wants to visually side "pass" or "fail" messages is different than... I don't know, say a CI job that needs to fail if the check inputs fails.

I haven't used the tool much in the wild, so the decision is yours.